actors/builtin/reward/reward_actor.go

```diff
@@ -68,8 +68,6 @@ type AwardBlockRewardParams struct {
 func (a Actor) AwardBlockReward(rt vmr.Runtime, params *AwardBlockRewardParams) *adt.EmptyValue {
 	rt.ValidateImmediateCallerIs(builtin.SystemActorAddr)
 	AssertMsg(params.Miner.Protocol() == addr.ID, "miner address must be ID-address (%d), was %d", addr.ID, params.Miner.Protocol())
-	AssertMsg(params.GasReward.Equals(rt.Message().ValueReceived()),
```

Comment:
The assertion that should be made instead is that the balance is greater than or equal to the params.GasReward.